pinot-transport/src/test/java/com/linkedin/pinot/transport/netty/NettyCloseChannelTest.java

```diff
@@ -86,6 +87,7 @@ public void testCloseClientChannel()
    * Client sends a request. Server closes the channel.
    */
   @Test
+  @Ignore
```

Comment:
Add a comment about why this is ignored

Reply:
Please use **@Test(enabled = false)**
Also. please add comment on why the test is ignored (copy-paste the commit message should be good enough)

Reply:
Thanks for the suggestions! Comments addressed.